scripts/vits/inflect/nano_v2/test.py

```
@@ -0,0 +1,138 @@
+        + " Whoever does not have two-thirds of his day for himself, "
+        + "is a slave, whatever he may be: a statesman, a businessman, "
+        + "an official, or a scholar."
+    )
+
+    model = OnnxModel(model="model.onnx")
+    start = time.time()
+
+    tokens = phonemize_espeak(text, "en-us")
+    #  print(text)
+    #  print(tokens)
+    tokens = sum(tokens, [])  # flatten
```

Ruff (0.15.21)

[warning] Avoid quadratic list summation

Replace with `functools.reduce`

(RUF017)